solutions/observability/apm/index.md

```diff
@@ -26,4 +26,4 @@ Metrics are another vital source of information when debugging production system
 
 ## Give Elastic APM a try [give_elastic_apm_a_try]
 
-Use [Get started with application traces and APM](/reference/fleet/get-started-managed-apm-server.md) to quickly spin up an APM deployment. Want to host everything yourself instead? See [Get started](/solutions/observability/apm/get-started.md).
\ No newline at end of file
+Use [Get started with application traces and APM](/solutions/observability/apm/apm-server-fleet-managed.md) to quickly spin up an APM deployment. Want to host everything yourself instead? See [Get started](/solutions/observability/apm/get-started.md).
```

Comment:
I would leave this link as it is: the aim of #2310 was to focus the APM main get started on Serverless, and pointing them to APM Server with Fleet breaks that.